index.md

```diff
@@ -3,6 +3,42 @@ title: Документация Elixir
 next_page: basic-types
 ---
 
+Содержание тестовое:
+* [Базовые типы](/docs/basic-types.html)
+* [Базовые операторы](/docs/basic-operators.html)
+* [Сопоставление с образцом](/docs/pattern-matching.html)
+* [Операторы ветвления](/docs/case-cond-and-if.html)
+* [Двоичные данные, строки и списки символов](/docs/binaries-strings-and-char-lists.html)
+* [Списки с ключевыми словами и словари](/docs/keywords-and-maps.html)
+* [Модули и функции](/docs/modules-and-functions.html)
+* [Рекурсия](/docs/recursion.html)
+* [Перечисления и потоки](/docs/enumerables-and-streams.html)
+* [Процессы](/docs/processes.html)
+* [Ввод/вывод и файловая система](/docs/io-and-the-file-system.html)
+* [alias, require и import](/docs/alias-require-and-import.html)
+* [Атрибуты модулей](/docs/module-attributes.html)
+* [Структуры](/docs/structs.html)
+* [Протоколы](/docs/protocols.html)
+* [Списковые выражения](/docs/comprehensions.html)
+* [Сигилы](/docs/sigils.html)
+* [try, catch и rescue](/docs/try-catch-and-rescue.html)
+* [Спецификации типов и поведения](/docs/typespecs-and-behaviours.html)
+* [Библиотеки Эрланга](/docs/erlang-libraries.html)
+* [Куда двигаться дальше](/docs/where-to-go-next.html)
+* [Введение в Mix](/docs/introduction-to-mix.html)
+* [Агент](/docs/agent.html)
+* [GenServer](/docs//genserver.html)
```

Comment:
Double slash in `/docs//genserver.html`

Reply:
This is completly wrong url if to be exact.